src/legacy/ui/public/vis/editors/default/_sidebar.scss

```diff
@@ -225,3 +219,7 @@
   margin-top: $euiSizeS;
   margin-bottom: $euiSizeS;
 }
+// prevent hiding a part of the border of accordion caret icon on focus
+.visEditorSidebar__aggGroup .euiAccordion__button {
```

Comment:
This actually needs to be fixed on the EUI side. You can remove this from here, and I'll make a note in EUI.

Reply:
Removed, thank you.